src/Psalm/Internal/PhpVisitor/CloningVisitor.php

```diff
@@ -22,9 +23,9 @@ public function enterNode(Node $node): Node
                 'comments',
                 array_map(
                     /**
-                     * @return \PhpParser\Comment
+                     * @return Comment
                      */
-                    function (\PhpParser\Comment $c): \PhpParser\Comment {
+                    function (Comment $c): Comment {
```

Comment:
This docblock is actually useless, let's drop it.

Reply:
I think we should’t do now. AFAIK there are more of these and there’s a phpcs sniff for it. Plan was to add this sniff (``SlevomatCodingStandard.TypeHints.ReturnTypeHint``) and do an extra PR then

Reply:
Ok then.